src/core/hiwire.c

```diff
@@ -67,17 +68,21 @@ EM_JS_NUM(int, hiwire_init, (), {
 
   Hiwire.new_value = function(jsval)
   {
-    // Should we guard against duplicating standard values?
-    // Probably not worth it for performance: it's harmless to occasionally
-    // duplicate. Maybe in test builds we could raise if jsval is a standard
-    // value?
```

Comment:
Could you add an inline comment explaining how it's handled now?

Reply:
Added a couple of comments explaining how we handle it now.